src/lago-expression/expression_js.d.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
Are those disabling auto-generated or you added them manually? Just curious if we can inline them at the correct place if only one line is concerned, of if it's a global failure

Reply:
I didn't add anything by hand in the `lago-expression` folder. It's all generated by `wasm-pack`.

Reply:
ok then!